### DISCUSSION TITLE
请问有训练自动生成 formily schema 的规划吗？

### DISCUSSION THREAD

**Discussion by BBSQQ** [alibaba/formily#4132] [Ideas]
输入：可以是设计稿、PRD，最好可以支持对话调整。
输出：formily schema

很需要，求。私聊可以戳👉： [dingtalk://dingtalkclient/action/sendmsg?dingtalk_id=p05ejgv](dingtalk://dingtalkclient/action/sendmsg?dingtalk_id=p05ejgv)

**Comment by liwei511** [comment]
有，太有了

**Reply by iworkerweb** [reply]
请问下 有了吗

**Comment by lijie1024512** [comment]
这个做了没？最近已经在尝试了

**Comment by flashtony2005** [comment]
A2UI
